src/Pos/Explorer/Web/Server.hs

```diff
@@ -121,23 +122,22 @@ getBlockTxs (fromCHash' -> h) (fromIntegral -> lim) (fromIntegral -> off) = do
         topsortTxsOrFail withHash txs
 
 getAddressSummary :: ExplorerMode m => CAddress -> m CAddressSummary
-getAddressSummary cAddr = cAddrToAddr cAddr >>= \case
+getAddressSummary cAddr = cAddrToAddr cAddr >>= \addr -> case addr of
```

Comment:
There is an ghc extension that allows to do `case of` just for this case. I think it is called `LabldaCase` and its already on.

https://www.schoolofhaskell.com/school/to-infinity-and-beyond/pick-of-the-week/guide-to-ghc-extensions/basic-syntax-extensions#lambdacase